basics/install.md

```diff
@@ -1,267 +0,0 @@
-## Installing GrimoireLab Python modules
```

Comment:
Sorry I forgot to post this comment in the earlier review. I would let this file to stay too, for some more time till we port it. This contains the information about building grimoirelab and it might be useful.

Reply:
ok. I updated the pr.